source/billing-data.txt

```diff
@@ -6,48 +6,138 @@ Billing Data
+On this page, learn how to use built-in {+service+} tools and
+{+service+} billing data to track, manage, and optimize your cloud
+spending.
 
-      .. include:: /includes/shared-settings-clusters-devtest.rst
-
-   .. tab:: Staging and Prod Environments
-      :tabid: stagingprod
-
-      .. include:: /includes/shared-settings-clusters-stagingprod.rst
-
-.. tabs::
-
-   .. tab:: CLI
-      :tabid: cli
-
-      Content here
-
-   .. tab:: Terraform
-      :tabid: Terraform
-
-      Content here
+{+service+} Features and Best Practices for Billing Data
+--------------------------------------------------------
 
+Billing Dashboards
+~~~~~~~~~~~~~~~~~~
+
+|charts| integreates seemlessly with {+service+}. You can visualize your
+billing data in a billing dashboard to help you optimize your |service|
+spending. Billing dashboards contain prebuilt charts that help you
+monitor your {+service+} usage in an organization across different
+categories and periods of time.
+
+.. figure:: /includes/images/atlas-billing-dashboard.png
+   :figwidth: 750px
+   :alt: Atlas Billing Dashboard example.
+
+To create or manage a billing dashboard, see :ref:`billing-dashboards`.
+
+Apply Resource Tags
+~~~~~~~~~~~~~~~~~~~
+
+The {+atlas-admin-api+} provides a |rest|\ful interface that allows you
+to retrieve all your pending invoices with the :oas-atlas-op:`Return All
+Pending Invoices for One Organization </listPendingInvoices>` |api|
+endpoint. The endpoints response body contains the
+``results.lineItems.tags`` field, which you can apply to your resources.
+
+Resource tags can help you categorize your spending and allow you to
+feed the |api| data to reports and external FinOps tools they already
+use. To learn more, see :ref:`configure-resource-tags`.
+
+Billing Quota Management
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+|service| offers real-time notifications when spending nears or exceeds
+set thresholds. You can customize these alerts to be project-specific or
+team-specific, allowing for precise monitoring and control over
+different areas of cloud expenditure. To configure billing alerts, see
+:ref:`configure-alerts`.
+
+Resource Tagging
+~~~~~~~~~~~~~~~~
+
+|service| uses resource tagging for precise cost allocation by
+categorizing resources according to departments, projects, or cost
+centers. You can also group and analyze tagged resources in financial
+reports, providing a clear and organized view of how various teams or
+projects contribute to overall cloud spending. To learn more, see
+:ref:`configure-resource-tags`.
+
+Cross-Organization Billing
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+|service| allows you to share a billing subscription across many
+organizations and to pay a single invoice for them. 
+
+After you configure a paying organization, you pay invoices for the paying
+organization that include a list of charges incurred for all linked
+organizations. To learn more, see :ref:`cross-org-billing`.
+
+Monthly Cost Visualization
+~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Can you make this change for the other two headings below as well? (change to "Review Your...")

```suggestion
Review Your Monthly Cost Visualization
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```